knotx-core/knotx-repository/src/main/java/com/cognifide/knotx/repository/RepositoryConfiguration.java

```diff
@@ -149,6 +154,14 @@ public void setType(RepositoryType type) {
             this.type = type;
         }
 
+        public JsonObject getClientOptions() {
+            return clientOptions;
+        }
+
+        public void setClientOptions(JsonObject clientOptions) {
```

Comment:
I don't see clientOptions are being set anywere except unit test. Looks like you missed passing it from verticle.